uninstall.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	apt "github.com/arduino/go-apt-client"
+	"github.com/docker/docker/api/types"
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/kardianos/osext"
+	"github.com/spf13/viper"
+	"golang.org/x/net/context"
+)
+
+// Uninstall removes all service installed and creates script to
+// removing application from machine
+func (s *Status) Uninstall(client mqtt.Client, msg mqtt.Message) {
+	data := "OK"
+
+	var err error
+
+	err = removeSketches(s)
+	if err != nil {
+		panic(err)
+	}
+
+	err = removeCerts(s)
+	if err != nil {
+		panic(err)
+	}
+
+	err = removeContainers(s)
+	if err != nil {
+		panic(err)
+	}
+
+	err = removeImages(s)
+	if err != nil {
+		panic(err)
+	}
+
+	err = removeNetworkManager()
+	if err != nil {
+		panic(err)
+	}
+
+	err = generateScriptUninstall()
+	if err != nil {
+		panic(err)
+	}
+
+	s.SendInfo(s.topicPertinence+"/status/uninstall", string(data))
+}
+
+func removeSketches(s *Status) error {
+	folder, err := getSketchFolder(s)
+	if err != nil {
+		return err
+	}
+
+	_, err = os.Stat(folder)
+	if err != nil {
+		return nil
+	}
+
+	err = os.RemoveAll(folder)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	if err := os.RemoveAll(folder); err != nil {
		return err
	}
```
`os.RemoveAll` won't return any errors if `folder` does not exist: https://golang.org/pkg/os/#RemoveAll